openbsd/libs.md

```diff
@@ -0,0 +1,116 @@
+# Instructions for obtaining EFI related libraries on OpenBSD
+
+Read [README.md](./README.md) first for general information. Note that building
+ports requires system with installed X11 parts of base system, you can load
+`bsd.rd` from `boot >` prompt, pick "Update" and install the missing packages if
+needed.
+
+The plan is:
+ * install kernel with support for EFI variables
+ * prepare environment for building ports
+ * build and install `efivar` port
+ * verify that it works
+
+## Install kernel with support for EFI variables
+
+Follow steps outlined in [efivars.md](./efivars.md) for this.
+
+## Preparing for using ports
+
+To prepare using ports as a regular user `user` perform these steps as `root`:
+
+```
+cat > /etc/mk.conf <<EOF
+WRKOBJDIR=/usr/obj/ports
+DISTDIR=/usr/distfiles
+PACKAGE_REPOSITORY=/usr/packages
+EOF
+
+mkdir /usr/distfiles /usr/packages
+chown user:wobj /usr/distfiles /usr/packages/
+
+# to be able to use ports as a regular user which is part of `wheel` group
+chmod -r g+w /usr/ports/
+```
+
+## Building and installing the port
+
+As a regular user assuming `doas` is configured:
+
+```
+git clone --depth 1 -b efivar https://github.com/3mdeb/ports /usr/ports
+
+cd /usr/ports/devel/efivar
+# build the port as a regular user
+make package
```

Comment:
@SergiiDmytruk I fails for me with:

```
[/usr/ports/devel/efivar]$ make package
===>  Checking files for efivar-38
>> Fetch https://github.com/3mdeb/efivar/archive/bsd-v1/efivar-38.tar.gz
>> Size does not match for efivar-38.tar.gz
>> Fetch https://ftp.openbsd.org/pub/OpenBSD/distfiles/efivar-38.tar.gz
ftp: Error retrieving https://ftp.openbsd.org/pub/OpenBSD/distfiles/efivar-38.tar.gz: 404 Not Found
>> Fetch https://ftp.usa.openbsd.org/pub/OpenBSD/distfiles/efivar-38.tar.gz
ftp: Error retrieving https://ftp.usa.openbsd.org/pub/OpenBSD/distfiles/efivar-38.tar.gz: 404 Not Found
>> Fetch https://ftp.fr.openbsd.org/pub/OpenBSD/distfiles/efivar-38.tar.gz
ftp: Error retrieving https://ftp.fr.openbsd.org/pub/OpenBSD/distfiles/efivar-38.tar.gz: 404 Not Found
*** Error 1 in . (/usr/ports/infrastructure/mk/bsd.port.mk:3209 '/usr/distfiles/efivar-38.tar.gz': @lock=efivar-38.tar.gz.dist;  /usr/bin/pe...)
*** Error 2 in . (/usr/ports/infrastructure/mk/bsd.port.mk:2485 '_internal-fetch': @cd /usr/ports/devel/efivar && PKGPATH=devel/efivar exec ...)
*** Error 2 in . (/usr/ports/infrastructure/mk/bsd.port.mk:2705 '/usr/obj/ports/efivar-38/.extract_done': @cd /usr/ports/devel/efivar && PKG...)
*** Error 2 in . (/usr/ports/infrastructure/mk/bsd.port.mk:2143 '/usr/packages/amd64/all/efivar-38.tgz': @cd /usr/ports/devel/efivar && PKGP...)
*** Error 2 in . (/usr/ports/infrastructure/mk/bsd.port.mk:2632 '_internal-package': @case X${_DEPENDS_CACHE} in  X) _DEPENDS_CACHE=$( mktem...)
*** Error 2 in /usr/ports/devel/efivar (/usr/ports/infrastructure/mk/bsd.port.mk:2611 'package': @lock=efivar-38;  export _LOCKS_HELD=" efiv...)
```

Reply:
I've added `make makesum` to instructions. The error is because I updated the fork, but not `distinfo` in the package, and that can happen again as fixes are added.

Reply:
@SergiiDmytruk This fixed this problem